lab-yunjoo/lib/note-crud.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const Note = require('../model/note');
+const AppErr = require('../lib/app-error');
+const debug = require('debug')('rito:note-crud');
+
+exports.createNote = function(reqBody){
+  return new Promise((resolve, reject) =>{
+    if(!reqBody.content)
+      return reject(AppErr.error400('note require\'s content'));
+    if(!reqBody.name)
+      return reject(AppErr.error400('note require\'s name'));
+
+    reqBody.timestamp = new Date();
+    const note = new Note(reqBody);
+    note.save() //save on 'note' objects, not 'Note' constructor
+    .then(resolve)
+    .catch(reject);
+  });
+};
+
+exports.fetchNoteByNoteId = function(id){
+  debug('fetchNote');
+  return new Promise((resolve, reject) =>{
+    Note.findOne({_id: id})
+    .then(resolve)
+    .catch(err => reject(AppErr.error404(err.message)));
+  });
+};
+
+exports.updateNote = function(id, data){
+  debug('updateNote');
+  return new Promise((resolve, reject)=> {
+    if(!id){
```

Comment:
All three of these if statements are identical. Is there a reason you didn't combine them?